app/views/layouts/application.html.erb

```diff
@@ -14,8 +14,9 @@
       </button>
       <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
         <div class="navbar-nav">
-          <a class="nav-item nav-link active" href="/dashboard">Dashboard<span class="sr-only">(current)</span></a>
```

Comment:
This class has extra stuff  (active etc) - is that necessary? Noticed it's not below in the new one.